assets/js/app.js

```
@@ -1249,6 +1249,233 @@ Hooks.StripePaymentElements = {
+              city = component.long_name;
+            } else if (component.types.includes('administrative_area_level_1')) {
+              state = component.long_name;
+            } else if (component.types.includes('country')) {
+              country = component.long_name;
+            }
+          }
+        }
+        
+        // Get coordinates
+        let lat = null, lng = null;
+        if (place.geometry && place.geometry.location) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)